XWIKI-10731: Make Create Wiki responsive

gabrielasmeria opened the pull request.

evalica commented on xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/CreateWiki.xml:
```diff
@@ -191,113 +191,115 @@
+          ### WIZARD FIRST HALF ###
+          &lt;div class = "col-xs-12 col-sm-6 col-md-6"&gt;
+            ### PRETTY NAME ###
             &lt;div&gt;
```
is this div needed?

evalica commented on xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/CreateWiki.xml:
```diff
@@ -191,113 +191,115 @@
-              &lt;/dd&gt;
-            &lt;/dl&gt;
+            ### WIKI NAME ###
+            &lt;div&gt;
```
needed?

evalica commented on xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/CreateWiki.xml:
```diff
@@ -191,113 +191,115 @@
                 &lt;/dt&gt;
                 &lt;dd&gt;
-                  &lt;input size="40" id="wikialias" name="wikialias" type="text"  value="$!escapetool.xml($request.wikialias)"/&gt;
+                  &lt;input size="60" id="prettyname" name="wikiprettyname" type="text" value="$!escapetool.xml($request.getParameter('wikiprettyname'))"/&gt;
```
We could use the same size for input type text. In http://platform.xwiki.org/xwiki/bin/view/DevGuide/VerticalForms, the size is recommended to '60'

evalica commented on xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/CreateWiki.xml:
```diff
@@ -191,113 +191,115 @@
+            &lt;/div&gt;
+            ### WIKI ALIAS ###
+            #if(!$services.wiki.isPathMode())
+              &lt;div&gt;
```
needed?

evalica commented on xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/CreateWiki.xml:
```diff
@@ -191,113 +191,115 @@
+              &lt;/div&gt;
+            #end
+            ### DESCRIPTION ###
+            &lt;div&gt;
```
again?

evalica commented on xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/CreateWiki.xml:
```diff
@@ -328,111 +330,113 @@
+          ### WIZARD FIRST HALF ###
+          &lt;div class="col-xs-12 col-sm-6 col-md-6"&gt;
+            ### OWNER ###
+            &lt;div&gt;
```
?

evalica commented on xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/CreateWiki.xml:
```diff
@@ -191,113 +191,115 @@
         &lt;input type="hidden" name="userScope" value="$!escapetool.xml($request.userScope)" /&gt;
         #addMembersAsHiddenInput()
-        ### WIZARD FIRST HALF ###
-        &lt;div class="half"&gt;
```
Just curious, how does this look on Colibri?

evalica commented:
So after thinking more about this issue, and thanks @Enygma2002 for reminding me, `.half` is extended in Flamingo in `grid.less`, see https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/grid.less#L12

So in theory, the layout should already be responsive, based on the backwards compatibility rule. 
So, actually the solution for this issue is: 
- do not change the .half classes, since they should work;
- but see what causes the different `width`. And the problem is that this was manually set in `skin/resources/uicomponents/wizard/wizard.css` to 48%. So we should remove those manually overrides of the skin defaults. 

The comments about extra `div`s and `size`s are still relevant and we should profit the occasion to fix them. WDTY?

evalica commented:
Btw, `wizard.css` should be looked at to see if there are other deprecated rules.

gabrielasmeria commented:
I didn't add those divs or change those sizes, I just moved the code from one place to another, that's why it looks like I added them. 
I tested to see if is still responsive when removing width: 48% and it is. So, I will close this PR and make a much cleaner one.
Thanks!
